Helper/Vault.php

```diff
@@ -369,32 +235,7 @@ private function getVaultCardToken(OrderPaymentInterface $payment, array $additi
         return $paymentToken;
     }
 
-    /**
-     * @param array $additionalData
-     * @return bool
-     */
-    private function validateAdditionalData(array $additionalData)
-    {
-        if (empty($additionalData)) {
-            return false;
-        }
-
-        foreach (self::ADDITIONAL_DATA_ERRORS as $key => $errorMsg) {
-            if (empty($additionalData[$key])) {
-                $this->adyenLogger->error($errorMsg);
-                return false;
-            }
-        }
-
-        return true;
-    }
-
-    /**
-     * @param $expirationDate
-     * @return string
-     * @throws Exception
-     */
-    private function getExpirationDate($expirationDate)
+    private function getExpirationDate($expirationDate): string
```

Comment:
Shall we add type declaration to `$expirationDate` argument? So that, we can remove `string` casting from `explode` function below.